code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -1001,7 +869,7 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/inflatable_dispenser(src)
 	// Medical
 	src.modules += new /obj/item/device/healthanalyzer(src)
-	src.modules += new /obj/item/reagent_containers/borghypo/medical(src)
+	src.modules += new /obj/item/reagent_containers/borghypo/rescue(src)
```

Comment:
I don't recommend fixing it like this. Rescue module only contains ("tricordrazine", "norepinephrine", "tramadol", "adrenaline".) That's missing a lot of critical medications in the normal medical borg synth ("bicaridine", "kelotane", "dylovene", "dexalin", "norepinephrine", "tramadol", "deltamivir", "thetamycin")

 I'd go into /reagent_containers/borghydro.dm and just add adrenaline to /borghypo/medical.  Unless it's intentional to drop these meds from the synth, in which case it should probably be changelogged.